fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -2330,7 +2330,19 @@ private String getAbsoluteUri(String baseUri, String relativeUri) {
             if (HTTPReturnPreference.REPRESENTATION.equals(FHIRRequestContext.get().getReturnPreference())) {
                 bundleEntryBuilder.resource(resource);
             } else if (HTTPReturnPreference.OPERATION_OUTCOME.equals(FHIRRequestContext.get().getReturnPreference())) {
-                bundleEntryBuilder.resource(operationOutcome);
+                OperationOutcome responseEntryOutcome = (OperationOutcome)responseEntry.getResource();
+                if (responseEntryOutcome != null && !responseEntryOutcome.equals(FHIRUtil.ALL_OK)) {
```

Comment:
Since OperationOutcome overrides equals, I think this is OK.  we're not comparing two pointers necessarily. 

Confirming (I also double checked this)